docs/using-getting-started.asciidoc

```diff
@@ -0,0 +1,283 @@
+
+| ecs.version
+| 1.5.0
+
+// ==============================================================
+
+| event.module
+| apache
+
+// ===============================================================
+
+| event.dataset
+| apache.access
+
+// ==============================================================
+
+| event.kind
+| event
+
+// ===============================================================
+
+| event.category
+| [ "web", "access" ]
+
+// ==============================================================
+
+| event.outcome
+| success
+
+|=====
+
+Lastly, existing field values can be interpreted or enriched using a processor, with the results
+populating additional fields in the final event.
+
+* The `user_agent` processor extracts details from the original user agent string, `user_agent.original`.
+* IP fields like `source.ip` can provide enrichment using the `geopip` processor to add information about the
+  location and autonomous system number (ASN) associated with an IP address.
+* The `registered domain` processor reads a field containing a hostname and writes the registered domain to
+  another field
+* Event collectors, such as https://www.elastic.co/guide/en/beats/libbeat/current/beats-reference.html[Beats], can enrich
+  each event with metadata from the machine's hosting provider (cloud) and/or from the host machine (host).
+
+Here are some examples of additional fields processed by metadata or parser processors.
+
+[options="header"]
+|=====
+| Field Name | Value | Processor
+
+// ==============================================================
+
+| host.architecture
+| x86_64
+| `add_host_metadata`
+
+// ==============================================================
+
+| host.hostname
+| mbp.example.com
+| `add_host_metadata`
+
+// ==============================================================
+
+| host.ip
+| [ "192.168.1.100" ]
+| `add_host_metadata`
+
+// ==============================================================
+
+| host.os.family
+| darwin
+| `add_host_metadata`
+
+// ==============================================================
+
+| host.os.kernel
+| 19.4.0
+| `add_host_metadata`
+
+// ==============================================================
+
+| host.os.name
+| Mac OS X
+| `add_host_metadata`
+
+// ==============================================================
+
+| host.os.version
+| 10.15.4
+| `add_host_metadata`
+
+// ==============================================================
+
+| user_agent.name
+| Chrome
+| `user_agent`
+
+// ==============================================================
+
+| user_agent.os.full
+| Mac OS X 10.15.4
+| `user_agent`
+
+// ==============================================================
+
+| user_agent.os.name
+| Mac OS X
+| `user_agent`
+
+// ==============================================================
+
+| user_agent.os.version
+| 10.15.4
+| `user_agent`
+
+// ==============================================================
+
+| user_agent.os.version
```

Comment:
That's a Chrome version! 😂 

```suggestion
| user_agent.version
```

![Laser Eyes!](https://webmat.keybase.pub/funny-stuff/dalai_lasers.gif)

Reply:
😂 It also was missing the final column and wasn't showing up in the rendered doc either! Cleaned up on the latest commit.